content/blog/2024/11/04/2024-11-04-jenkins-tutorial-revamp.adoc

```diff
@@ -0,0 +1,273 @@
+The entire process takes less than a minute - a significant improvement over traditional setup methods.
+Since we've streamlined the authentication process, you can log in immediately using the credentials admin/admin and begin your Jenkins journey.
+
+video::oEcreaxR6MQ[youtube,width=839]
+
+Expanding the window reveals the full Jenkins interface.
+Upon logging in with the tutorial credentials (`admin`/`admin`), you'll immediately notice several key features: a pre-configured job and an active agent visible in the left sidebar.
+This job, while simple, serves an important purpose—it demonstrates that both your Jenkins controller is operational and the agent is properly connected.
+Navigating to "_`Manage Jenkins`_" shows that all plugins are current, eliminating any need for initial administrative setup.
+The bottom right corner displays the current Jenkins LTS version (`2.462.2` at the time of recording), confirming you're working with the latest stable release.
+This streamlined environment allows users to focus entirely on learning Jenkins, without getting caught up in configuration details.
+The entire setup process, from start to finish, takes less than two minutes and requires no local installation—perfectly aligned with our goal of making Jenkins more accessible to everyone.
+
+video::pEaWpCm1xLU[youtube,width=839]
+
+The GitPod terminal output illustrates another key innovation: our implementation of Docker Compose profiles for various programming languages and technologies.
+This feature exemplifies our commitment to simplicity—switching between different technology stacks, whether Maven, Python, or Node.js, requires only a single command: `docker compose --profile maven up -d`.
+The system handles all the complexity behind the scenes, delivering a seamless experience that lets developers focus on learning Jenkins rather than wrestling with configuration.
+
+This streamlined approach reflects our broader commitment to inclusivity.
+Through multi-architecture support and browser-based accessibility via GitPod, we've created an environment where developers of all experience levels and technical resources can engage with Jenkins effectively.
+
+=== 3. Flexible Configurations
+
+Docker Compose profiles allow easy switching between different tech stacks. Whether you're working with Maven, Python, or Node.js, you can start the appropriate environment with a simple profile switch.
+
+=== 4. Up-to-date and Secure
+
+By eliminating Docker in Docker and implementing automated updates, we've made the tutorials more secure and easier to maintain.
+
+== The Impact: Before and After
+
+The impact of these changes has been significant and far-reaching:
+
+1. *Increased adoption*: Our Docker images have been downloaded over 26,000 times since launch. We've seen multiple forks of the project, with people using it to create their own Jenkins instances for other technologies.
```

Comment:
```suggestion
1. *Increased adoption*: Our Docker images have been downloaded over 26,000 times since launch.
We've seen multiple forks of the project, with people using it to create their own Jenkins instances for other technologies.
```
Single line per sentence suggestion